lib/rules/no-unsupported-features/node-builtins.js

```diff
@@ -16,10 +16,15 @@ const {
     NodeBuiltinModules,
 } = require("../../unsupported-features/node-builtins.js")
 
+/**
+ * @typedef {Object} TraceMap
```

Comment:
```suggestion
 * @typedef TraceMap
```

When using `@property` no type is needed in `@typedef`, its implied that its an object